.changeset/quiet-lies-peel.md

```diff
@@ -0,0 +1,12 @@
+---
+'@leafygreen-ui/banner': patch
+'@leafygreen-ui/icon-button': patch
+'@leafygreen-ui/mongo-nav': patch
+'@leafygreen-ui/radio-box-group': patch
+'@leafygreen-ui/select': patch
+'@leafygreen-ui/tabs': patch
+'@leafygreen-ui/text-input': patch
+'@leafygreen-ui/toast': patch
+---
+
+Updates colors across components to better comply with WCAG contrast ratio standards
```

Comment:
We should make specific changesets for unrelated changes and call out specifically what's being changed (I consider separate contrast updates to be unrelated changes).

Reply:
They won't appear as related when we release the changesets, but if that's the only thing that changed the note should be able to be the same in each individual changelog

Reply:
Banner: 
IconButton: https://jira.mongodb.org/browse/PD-1040
MongoNav: https://jira.mongodb.org/browse/PD-1042
RadioBoxGroup: https://jira.mongodb.org/browse/PD-1050
Select:
Tabs:
TextInput: https://jira.mongodb.org/browse/PD-1048
Toast:

Reply:
The note refers to "colors across components" so it won't appear specific to each component.

Reply:
Additionally, we're not telling people where to expect the change. E.g. is it in a focus state, the static state, etc/

Reply:
Broke these out!